_posts/payalpatel/2016-02-03-Ch5Exercises.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: payalpn
+title: "Payal's Chapter 5 Loops Exercises"
+---
+
+**Chapter 5, Exercise 1:**
+
+<iframe src="https://trinket.io/embed/python/0c8682b715" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Chapter 5, Exercise 2:**
+
+<iframe src="https://trinket.io/embed/python/5a5b5b49bc" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Reflection:**
+
+At first I thought both of these exercises were difficult.  I was having trouble starting, so in order to deal with this I split up the problems into divisible and more manageable tasks.  I then arranged them in an order that I thought made logical sense (in terms of completing each of these smaller subtasks).  For me this helped a lot—I was able to focus on smaller tasks and then piece them together once I had finished the components.  
+
+In the Chapter 5 reading, counting/summing/maximum/minimum loops were created using for loops.  I thought about using this, however, it seemed simpler to use the while loop and try/except.  To determine the maximum and minimums, I created an empty list which would then store numbers entered in by the user.  After that I calculated the maximum and minimum of this list and stored them into maximum and minimum variables, respectively.  
```

Comment:
Make sure you have a sense of how to solve this without the list as well- the list functions are doing a lot of work for you.